yarn-project/p2p/src/bootstrap/bootstrap.ts

```diff
@@ -32,22 +32,17 @@ export class BootstrapNode implements P2PBootstrapApi {
    */
   public async start(config: BootnodeConfig) {
     const { udpListenAddress, udpAnnounceAddress } = config;
-
-    const peerIdPrivateKey = await getPeerIdPrivateKey(config, this.store);
-    const peerId = await createLibP2PPeerIdFromPrivateKey(peerIdPrivateKey);
-    this.peerId = peerId;
-    const enr = SignableENR.createFromPeerId(peerId);
-
     const listenAddrUdp = multiaddr(convertToMultiaddr(udpListenAddress, 'udp'));
 
     if (!udpAnnounceAddress) {
       throw new Error('You need to provide a UDP announce address.');
     }
 
-    const publicAddr = multiaddr(convertToMultiaddr(udpAnnounceAddress, 'udp'));
-    enr.setLocationMultiaddr(publicAddr);
+    const peerIdPrivateKey = await getPeerIdPrivateKey(config, this.store);
+    const peerId = await createLibP2PPeerIdFromPrivateKey(peerIdPrivateKey);
+    this.peerId = peerId;
```

Comment:
Heads up this is already being done inside `createBootnodeENR`, maybe we can extract the peerId from there?

Reply:
Refactored